pom.xml

```diff
@@ -470,6 +470,12 @@
             </plugins>
         </pluginManagement>
 
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```

Comment:
Hva tenker du med denne? 
Denne finnes i regelrepos for å kunne lagre regelversjon i fpsak/kalkulus - men det går på jus/fag/sporing. 
Ser ikke helt samme behovet her

Reply:
Vil vi ikke rulle ut nye versjoner av task til alle apps samtidig? Vi ønsker ikke ha apps som lagger bom/felles/task

Reply:
Vilje er en ting - men i praksis er det nesten en dagsverk om man skal ta alle 17 apper men en gang. 
Jeg har forenklet denne delen.